app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,16 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+  
+  def create
+    @order = Order.new(order_params,service_id: params[:service_id],client_id: current_user.id)
```

Comment:
if you change the routes and make `order` a nested resource to `service`, you can do something like:

```
@order = Service.find(params[:service_id]).orders.create(order_params, client_id: current_user.id)
```